test/test_view/test_assign_task_worker.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf8 -*-
+import json
+from mock import patch
+from test import with_context
+
+from default import db
+from factories import ProjectFactory, TaskFactory, UserFactory
+from helper import web
+from pybossa.repositories import ProjectRepository, TaskRepository, UserRepository
+
+project_repo = ProjectRepository(db)
+user_repo = UserRepository(db)
+task_repo = TaskRepository(db)
```

Comment:
Unused and can be removed. Also line #6 importing db can be removed that would resolve the test failures too it seems.